### PR TITLE
Factor out common modal dialog code into templates.

### PR DESCRIPTION
**Pull request by larspetrus**
Trying some Blaze refactoring.

### CODE REVIEW

File: templates/editEvents.html [hunk 1]
```diff
@@ -164,17 +164,14 @@ <h3 class="panel-title">
...
-                            </div>
+                            {{#cmHeader}}
+                              Are you sure you want to delete {{eventName eventCode}}: {{roundName lastRoundCode}}?
+                            {{/cmHeader}}
                             <div class="modal-body">
                               <p>It appears that this round contains results. If you delete it, those results will be lost.</p>
                             </div>
                             <div class="modal-footer">
-                              <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                              {{> cmCancelButton}}
```
Comment (jfly) [inline templates/editEvents.html]: cm or ccm?

**Comment by jfly** [comment]
Looks good to me! I see a couple of TODOs, are those something you want to deal with before merging? If not, lets create issues for them and then merge up. Also curious about the ccm vs cm prefix.

File: templates/editEvents.html (hunk 1 above)
Comment (larspetrus) [inline templates/editEvents.html]: I had a Hamletian struggle with that too. I probably got it wrong.

`cm` is meant to be short for `ccmModal`. Because `ccmModalCancelButton` it too wordy, and `ccmCancelButton` doesn't properly convey that it's part of the `ccmModal` "package".

Of course the `cm` prefix doesn't convey anything either unless I can show up in person and give this mini-lecture to everyone reading the code.

So... I'm OK with `ccm`, unless you have an even better suggestion. When you look it up you see what it's for.

Comment (jfly) [inline templates/editEvents.html]: OH! Gotcha. I'm fine with `ccmCancelButton` or `ccmModalCancelButton`. One crazy idea is to keep `cm` and rename `ccmModal` to `cm`. Whatever you think is best. As you said, it's easy to look these things up.

**Comment by larspetrus** [comment]
Those TODOs are modals I wanted to convert ccmModal, found hard, but still had hope.

For now they should just be removed.

**Comment by jfly** [comment]
Sounds good! Will merge as soon as the TODOs are resolved.

**Comment by jfly** [comment]
Thanks Lars!
